corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -274,6 +275,14 @@ def build_query_prompts(self):
                 kwargs['exclude'] = "true()"
             if prop.required:
                 kwargs['required'] = interpolate_xpath(prop.required)
+            if prop.validation:
+                kwargs['validation'] = [
+                    Validation(
+                        test=condition.xpath,
```

Comment:
Calling `interpolate_xpath` on this will add support for `#user`, etc.

Reply:
Thinking about it, a `#search` or `#search_input` shortcut would be pretty nice. Though it's be a little different from `#user` and `#case`, I'd think we'd want to support something like `#search/email`, which would expand to `instance('search-input:results')/input/field[@name='email']`.